app/src/test/java/com/github/se/polyfit/model/nutritionalInformation/MeasurementUnitTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.github.se.polyfit.model.nutritionalInformation
+
+import android.util.Log
+import io.mockk.every
+import io.mockk.mockkStatic
+import junit.framework.TestCase.assertEquals
+import kotlin.test.Test
+import kotlin.test.assertFailsWith
+import org.junit.Before
+
+class MeasurementUnitTest {
+  @Before
+  fun setup() {
+    mockkStatic(Log::class)
+    every { Log.e(any(), any()) } returns 0
+    every { Log.e(any(), any(), any()) } returns 0
```

Comment:
what do these contribute to the tests below? i can't tell where it is used/referenced

Reply:
Yeah so because this is a unit test and in the folder tests (not androidTest), it is not emulated in an actual android environment. So you need to mockk these functions in order for the test to compile